runtime/lua/vim/diagnostic.lua

```diff
@@ -2348,8 +2348,8 @@ end
 --- @class vim.diagnostic.setqflist.Opts
 --- @inlinedoc
 ---
---- Only add diagnostics from the given namespace.
---- @field namespace? integer
+--- Only add diagnostics from the given namespace(s).
+--- @field namespace? integer[]|integer
```

Comment:
No further changes are needed here besides changing the type as `vim.diagnostic.get()` already handles a single/multiple namespaces.